Gemfile

```diff
@@ -38,14 +38,16 @@ gem 'kaminari'
 gem 'caxlsx'
 gem 'caxlsx_rails'
 
+gem 'docx'
+
 gem 'httparty'
 gem 'github_api', '~> 0.18.2'
 gem 'active_model_serializers', '~> 0.10.13'
 gem 'money-rails', '~> 1.12'
 
 gem 'ransack', '~> 2.3'
 
-gem 'sidekiq', "< 7"
+gem 'sidekiq', "< 8"
```

Comment:
this is not necessary now and redis on heroku does not support yet

Reply:
ok, removing it.